scrapy/utils/conf.py

```diff
@@ -34,6 +34,16 @@ def _map_keys(compdict):
             _check_components(compdict)
             return {convert(k): v for k, v in six.iteritems(compdict)}
 
+    def _ensure_numeric_value(compdict):
+        for name, value in six.iteritems(compdict):
+            if value is None or isinstance(value, (int, float)):
+                continue
+            try:
+                compdict[name] = float(value)
```

Comment:
I misunderstood your words, I thought you were talking about an upcoming change in the way of defining components order after scrapy 1.3 :-P